### PR TITLE
feat(link): migrate link component

### PR DESCRIPTION
**Pull request by alexandereneroth**
Migrate link component

Fixes: [AB#2399](https://dev.azure.com/scaniadigitaldesignsystem/3d9eb9c6-0045-473d-bc1d-e842dd779200/_workitems/edit/2399)

### CODE REVIEW

**Review by mJarsater — APPROVED** [review]
Looks good to me

File: components/src/components/link/link.stories.ts [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+  ...Default.args,
+  disabled: true,
+};
+
+export const NoUnderline = Template.bind({});
+
+NoUnderline.args = {
+  ...Default.args,
+  noUnderline: true,
+};
```
Comment (mJarsater) [inline components/src/components/link/link.stories.ts:55]: Should this be a new story? Or would we want to streamline it and let the users view this via the controls.

Comment (alexandereneroth) [reply components/src/components/link/link.stories.ts:55]: I feel like there's value to displaying the different variations to the user up front.
Some examples:
- https://digitaldesign.scania.com/components/datepicker/code 
- https://chakra-ui.com/docs/components/link/usage
- https://mui.com/joy-ui/react-link/

Comment (mJarsater) [reply components/src/components/link/link.stories.ts:55]: Absolutely, I think this also is a question of how user actually use the site. Are they interacting with the controls or are they checking the stories.

**Review by theJohnnyMe — CHANGES_REQUESTED** [review]
Please leave only the "Default" story in this one. It is a very simple component. Thanks for fixing the disabled state too.

**Comment by alexandereneroth** [comment]
> Please leave only the "Default" story in this one. It is a very simple component. Thanks for fixing the disabled state too.

Removed the other stories. 
Also added a demo class with a font size so the text is not tiny.

File: components/src/components/link/link.stories.ts [hunk 2]
```diff
@@ -0,0 +1,48 @@
+import { Meta, StoryFn } from '@storybook/html';
+import clsx from 'clsx';
+import { formatHtmlPreview } from '../../utils/utils';
+
+const defaultArgs = {
```
Comment (mJarsater) [inline components/src/components/link/link.stories.ts:5]: Is this object needed? The default values are already present in the default export.

**Pull request merged**